types/example.pp

```diff
@@ -1,14 +1,14 @@
 type Patterndb::Example = Struct[
   {
-    program      => String[1],
+    program      => String,
     test_message => String[1],
     test_values  => Optional[
       Hash[
         String[1],
         Struct[
           {
-            type  => Optional[Enum['datetime', 'double', 'integer', 'json', 'string']],
-            value => String[1],
+            Optional[type]  => Enum['datetime', 'double', 'integer', 'json', 'string', 'null'],
```

Comment:
`type` is a keyword, and seems to confuse the parser now :upside_down_face:.

This will work:

```suggestion
            Optional['type'] => Enum['datetime', 'double', 'integer', 'json', 'string', 'null'],
```

Reply:
I replaced all wrong-old-style-optionals in latest commit